packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -54,6 +56,16 @@ const useMenuItems = (): NavItemProps[] => {
           isHeader: true,
         },
         { label: 'Edit profile', icon: <EditIcon />, href: '/account/profile' },
+        ...(!user?.isPlus
+          ? [
+              {
+                label: 'Upgrade to plus',
+                icon: <DevPlusIcon />,
+                href: plusUrl,
+                className: 'text-accent-bacon-default',
+              },
+            ]
+          : []),
```

Comment:
This one needs to toggle to become Manage Plus when you have it.
(Link to external, yet to be defined)

![Screenshot 2024-11-13 at 10 17 22](https://github.com/user-attachments/assets/a7df2f52-d619-46d8-aa64-8d863f2cee0c)

Reply:
Fixed